src/ows/src/main/java/org/geoserver/ows/Dispatcher.java

```diff
@@ -1151,6 +1151,10 @@ Service findService(String id, String ver, String namespace) throws ServiceExcep
         if (matches.size() > 1) {
             List vmatches = new ArrayList(matches);
 
+            if (isCiteCompliant() && version == null && "WFS".equals(id)) {
```

Comment:
I agree as well. That said, I feel like the way we have things set up right now, implementing this sort of thing is going to be a bit tricky.

If `isCiteCompliant()` is `false` for `ows`, is there any check we can do for running cite tests?